CHANGELOG.md

```diff
@@ -134,13 +134,13 @@
 
 - \[QUESTION\] kitchen-inspec formatter [\#147](https://github.com/inspec/kitchen-inspec/issues/147)
 - Executing inspec against a container from within the host [\#145](https://github.com/inspec/kitchen-inspec/issues/145)
-- Option to specify path to Inspec test with kitchen-inspec [\#139](https://github.com/inspec/kitchen-inspec/issues/139)
+- Option to specify path to InSpec test with kitchen-inspec [\#139](https://github.com/inspec/kitchen-inspec/issues/139)
```

Comment:
I ❤️ you so much for  these changes.

Reply:
I worked for a company that changed the case of their name to be all lowercase (so cool). I spent years slowly fixing it all.